src/main/java/com/pnu/dev/radioserviceapi/service/BannerApiServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.pnu.dev.radioserviceapi.service;
+
+import com.pnu.dev.radioserviceapi.mongo.Banner;
+import com.pnu.dev.radioserviceapi.repository.BannerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class BannerApiServiceImpl implements BannerApiService {
+
+    private BannerRepository bannerRepository;
+
+    @Autowired
+    public BannerApiServiceImpl(BannerRepository bannerRepository) {
+        this.bannerRepository = bannerRepository;
+    }
+
+    @Override
+    public Map<String, String> findBannersToShow() {
+        return bannerRepository.findAll().stream()
+                .filter(Banner::isShow)
```

Comment:
Or add `findAllByIsShow(boolean show)` method to Repository, but let it be

Reply:
Yes, sure. 
`List<Banner> findAllByShowTrue();` would be even better